lighthouse-treemap/app/styles/treemap.css

```diff
@@ -71,7 +96,35 @@ body {
 }
 
 .lh-treemap {
-  margin: 10px;
+  margin: 2px;
+  contain: content;
+}
+
+.lh-table {
+  overflow-y: hidden;
+}
+
+.tabulator {
+  /* Better default for unloaded portions of table. */
+  background-color: #f3f3f3;
+  contain: strict;
+}
+
+.lh-coverage-bar {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+.lh-coverage-bar--used {
+  background-color: green;
```

Comment:
didnt we want to match devtools' colors? because vision deficiencies..

their red: #ee442f
their green: #63acbe